deployment/common/common.yaml

```diff
@@ -22,4 +22,108 @@ spec:
   names:
     plural: ingressroutes
     kind: IngressRoute
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+            - routes
+          properties:
+            virtualhost:
+              properties:
+                fqdn:
+                  type: string
+                  pattern: ^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$
```

Comment:
Are uppercase chars not allowed in FQDNs?

Reply:
Yup they are allowed, just was going with the default that they are typically lowercase. Will update to follow the spec.